api/nodes/serializers.py

```diff
@@ -63,10 +40,12 @@ class NodeSerializer(JSONAPISerializer):
                                                             'public and private nodes. Administrators on a parent '
                                                             'node have implicit read permissions for all child nodes',
                               )
-    # TODO: finish me
 
-    class Meta:
-        type_ = 'nodes'
+    registration = ser.BooleanField(source='is_registration', read_only=True)
```

Comment:
@brianjgeiger Does removing `is_` follow convention for the rest of the API? Personally, I liked having the `is_` because it made clear which attributes are booleans.